08-service/README.md

```diff
@@ -159,7 +159,7 @@ $ minikube ip
 192.168.99.100
 ```
 
-Open a browser on `http://192.168.99.100/info`
+Open a browser on `http://192.168.99.100/health`
```

Comment:
The endpoint does not exist on 0.4.0 only on [0.5.0+](https://github.com/mhausenblas/simpleservice#info-050).